From: Simon McVittie <smcv@collabora.com>
Date: Mon, 28 Jun 2021 14:18:26 +0100
Subject: completions: Don't start bash completion with #!

bash completions are sourced, not executed, so this doesn't need to be
an executable script.

Forwarded: https://github.com/containers/bubblewrap/pull/439
Signed-off-by: Simon McVittie <smcv@collabora.com>
---
 completions/bash/bwrap | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/completions/bash/bwrap b/completions/bash/bwrap
index 4ca0c26..9389afd 100644
--- a/completions/bash/bwrap
+++ b/completions/bash/bwrap
@@ -1,5 +1,5 @@
-#!/bin/bash
-#
+# shellcheck shell=bash
+
 # bash completion file for bubblewrap commands
 #
 
@@ -67,3 +67,5 @@ _bwrap() {
 	return 0
 }
 complete -F _bwrap bwrap
+
+# vim:set ft=bash:
